content/en/contribute/index.mdx

```diff
@@ -190,3 +190,6 @@ We welcome ideas for improving our community resources. Submit a <Link to="/fund
 
 </Tout>
 
+## Our Contributor Community
+
+<Authors all/>
```

Comment:
There is some styling coming that should remove the block-ey feeling to all authors being displayed.

One thing that would be helpful is considering how many authors should display and then rotating them on some automatic frequency.